backend/app/services/parse_csv.py

```diff
@@ -0,0 +1,37 @@
+from app.models import Document, Corpus
+
+def parse_csv(csv_reader, title):
+    doc_list = []
+    for data in csv_reader:
+        new_doc = None
+        if Document.objects.filter(title=data['title']).count() == 0:  # If the data isn't already an instance in the Document model:
```

Comment:
Awesome job trying to anticipate the possibility of duplicate records in the database, but it's probably worth thinking more about. That is: filtering `Documents` based on title to find out if the document under iteration (here described as `data`) already exists in the database. It's possible that two separate `Document` instances might have the same title but not be the same text. This is kind of a fundamental problem: what makes a texts unique? Title? Title and author? Title and author and wordcount? That all words match? 

That seems like kind of a big deal to sort out, and beyond the scope of this feature add. I'd recommend we simply dispense with the conditional check here and just write a new `Document`. That is: let later us figure out how to answer "what makes a text unique?"